sqlsynthgen/main.py

```diff
@@ -28,7 +39,31 @@ def create_data(
     ssg_file: str = typer.Argument(...),
     num_rows: int = typer.Argument(...),
 ) -> None:
-    """Fill tables with synthetic data."""
+    """Populate schema with synthetic data
+    This CLI command generates synthetic data for
+    Python table structures, and inserts these rows
+    into a destination schema.
+
+    Also takes as input object relational model as represented
+    by file containing Python classes and its attributes.
+
+    Takes as input sqlsynthgen output as represented by Python
+    classes, its attributes and methods for generating values
+    for those attributes.
+
+    Final input is the number of rows required.
+
+    Example:
+        $ python sqlsynthgen/main.py create-data example_orm.py expected_ssg.py 100
```

Comment:
The CLI examples can all be:

```suggestion
        $ sqlsynthgen create-data example_orm.py expected_ssg.py 100
```

without the `python sqlsynthgen/main.py`